dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -416,22 +477,16 @@ public void ContentFallbackFalse_PageFallbackTrue_PageEnglishAndSpanish_ViewEngl
      * @throws Exception
      */
     @Test (expected = HTMLPageAssetNotFoundException.class)
-    public void ContentFallbackFalse_PageFallbackTrue_PageSpanish_ViewEnglish404_ViewSpanishContent2And3() throws Exception{
+    @UseDataProvider("cases")
+    public void ContentFallbackFalse_PageFallbackTrue_PageSpanish_ViewEnglish404_ViewSpanishContent2And3(final Container container, final Template template) throws Exception{
```

Comment:
I would add javadoc explaining better the given scenario and expected result

Reply:
this test is not new

Reply:
same here, if you know what the test does, please add the javadoc